tests/sitemap.test.js

```diff
@@ -18,9 +18,9 @@ const urlset = '<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" ' +
              'xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">'
 
 const dynamicUrlSet = '<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">'
-const xmlDef = '<?xml version="1.0" encoding="UTF-8"?>\n'
-const xmlPriority = '<priority>0.9</priority> '
-const xmlLoc = '<loc>http://ya.ru</loc> '
+const xmlDef = '<?xml version="1.0" encoding="UTF-8"?>'
```

Comment:
the lib doesn't have whitespace between tags. The tests have been updated to match. the output. Otherwise all tests pass and produce otherwise identical results.